draft-ietf-mls-protocol.md

```diff
@@ -2260,14 +2236,6 @@ Update for the leaf if there are no Removes. If there are multiple Add proposals
 for the same client, the committer again chooses one to include and considers
 the rest invalid.
 
-If there are multiple ReInit proposals, the committer MUST choose exactly one to
```

Comment:
Why remove this? I think we should at least specify that only one re-init proposal can be included in a commit.

Reply:
This discussion was moved to the ReInit proposal section

> If a ReInit proposal is included in a Commit, it MUST be the only proposal
referenced by the Commit.